src/app/app.component.html

```diff
@@ -580,4 +587,34 @@ <h3 class="modal-title">Change Password</h3>
       </button>
     </div>
   </clr-modal>
+  <clr-modal [(clrModalOpen)]="alertDeleteAccessCodeModal">
+    <h3 class="modal-title">Are you sure you want to delete?</h3>
+    <div class="modal-body">
+      <div class="alert alert-danger">
+        <div class="alert-items">
+          <div class="alert-item static" role="alert">
+            <div class="alert-icon-wrapper">
+              <cds-icon
+                class="alert-icon"
+                shape="exclamation-triangle"
+              ></cds-icon>
+            </div>
+            <span class="alert-text">
+              The access code will be deleted permantely!
```

Comment:
Use Plural here

Reply:
Good, I just fixed that, now the modal will display phrase in plural if the user selected multiple access codes to delete and in singular if the user delete just one access code.